src/Parser.php

```diff
@@ -7,17 +7,21 @@
 
 class Parser
 {
-    public static function getSchema()
+    public static function getSchema($exclude = [])
     {
-        return collect(DB::select('show tables'))->map(function ($row) {
-            $table = head($row);
+        return collect(DB::select('show tables'))
```

Comment:
Please add one more `map` to prevent the redundant and implicit `head($row)`  statements.